src/scripts/install.py

```diff
@@ -63,6 +63,19 @@ def log_level():
 
     return (options, args)
 
+def prepend_destdir(path):
+    """
+    Needed because os.path.join() discards the first path if the
+    second one is absolute, which is usually the case here. Still, we
+    want relative paths to work and leverage the os awareness of
+    os.path.join().
+    """
+    try:
+        destdir = os.environ['DESTDIR']
+        return os.path.join(destdir, path.lstrip('/'))
+    except KeyError as e:
+        return path
+
```

Comment:
Agreed about the empty prefix, even if rare. However, I am not sure why `DESTDIR` should not be legitimate on Windows (I've actually used that feature on Windows in a cross OS project before to copy the files into the right location for the installer-generator to pick them up). 

Windows converts `/` to `\` internally, so it should still work on windows jolly fine. IIRC it can even handle mixed slashes in a path. If you want to be extra sure, we can call `os.path.normpath()` on the result.